src/Components/PurchaseCard/PurchaseCard.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  Grid,
+  GridItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+
+const PurchaseCard = () => {
+  return (
+    <>
+      <Card>
```

Comment:
Same as above, extra return statement and no need for extra element
```suggestion
const PurchaseCard = () => (
  <Card>
```